<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QG_CadToolBarSelect</class>
 <widget class="QWidget" name="QG_CadToolBarSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>56</width>
    <height>336</height>
   </size>
  </property>
  <widget class="QToolButton" name="bBack" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnAll" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bAll" >
   <property name="geometry" >
    <rect>
     <x>28</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bSingle" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bContour" >
   <property name="geometry" >
    <rect>
     <x>28</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bWindow" >
   <property name="geometry" >
    <rect>
     <x>28</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnWindow" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnInters" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>104</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bInters" >
   <property name="geometry" >
    <rect>
     <x>28</x>
     <y>104</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bInvert" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>132</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bLayer" >
   <property name="geometry" >
    <rect>
     <x>28</x>
     <y>132</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bForward" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>160</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="iconSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../qcadguiqt.qrc" />
 </resources>
 <connections/>
</ui>
